projects/fsastorefrontlib/src/cms-components/my-account/inbox/inbox-tab/inbox-messages/inbox-messages.component.html

```diff
@@ -71,20 +72,27 @@
         <span class="checkbox-text fs-icon icon-ok"></span>
       </label>
     </div>
-    <div
-      class="col-8 text-center"
-      (click)="displayMobileGroups = !displayMobileGroups"
+    <ng-container
+      *ngIf="{
+        subject: defaultSortOrder,
+        content: defaultSortOrder,
+        sentDate: defaultSortOrder
+      } as sorts"
     >
-      {{ mobileGroupTitle }}
-    </div>
-    <div class="col-2">
-      <span
-        class="filter position-absolute fs-icon icon-triangle-bottom"
-      ></span>
-    </div>
+      <div
+        class="col-9 text-left ml-4 heading-dropdown"
+        (click)="displayMobileGroups = !displayMobileGroups"
+      >
+        <span>{{ mobileGroupTitle }}</span>
+        <span
+          class="fas fa-chevron-down icon-triangle-bottom"
```

Comment:
`.icon-triangle-bottom` seems redundant here

Reply:
It is needed here, because that class has styles for rotating icon.

Reply:
Then this would indicate that there is also our FSA icon present, and there isn't. It should be used in the right context. If you need rotation and other behavior that `icon-triangle-bottom` contains, then either make some unified class for them, or just simply add `fa-chevron-down` next to `.icon-triangle-bottom` in scss rule.

Reply:
And as a side note, I don't think `dropdowns`(select elements) usually have rotating arrow...

Reply:
Ok, then I remove that style.